controllers/resource/upserter_test.go

```diff
@@ -163,59 +161,14 @@ func Test_upserter_UpsertDoguDeployment(t *testing.T) {
 }
 
 func Test_upserter_UpsertDoguPVCs(t *testing.T) {
-	t.Run("fail when creating a reserved pvc", func(t *testing.T) {
-		// given
-		doguResource := readLdapDoguResource(t)
-		dogu := readLdapDogu(t)
-		dogu.Volumes = nil
-
-		generator := mocks.NewDoguResourceGenerator(t)
-		generator.On("CreateReservedPVC", doguResource).Return(nil, assert.AnError)
-		upserter := upserter{
-			generator: generator,
-		}
-
-		// when
-		_, err := upserter.UpsertDoguPVCs(context.Background(), doguResource, dogu)
-
-		// then
-		require.ErrorIs(t, err, assert.AnError)
-	})
-	t.Run("fail when upserting a reserved pvc", func(t *testing.T) {
-		// given
-		doguResource := readLdapDoguResource(t)
-		dogu := readLdapDogu(t)
-		dogu.Volumes = nil
-
-		mockClient := extMocks.NewK8sClient(t)
-		key := doguResource.GetObjectKey()
-		key.Name = doguResource.GetReservedPVCName()
-		mockClient.On("Get", mock.Anything, key, &v1.PersistentVolumeClaim{}).Return(assert.AnError)
-
-		generator := mocks.NewDoguResourceGenerator(t)
-		generator.On("CreateReservedPVC", doguResource).Return(readLdapDoguExpectedReservedPVC(t), nil)
-		upserter := upserter{
-			client:    mockClient,
-			generator: generator,
-		}
-
-		// when
-		_, err := upserter.UpsertDoguPVCs(context.Background(), doguResource, dogu)
-
-		// then
-		require.ErrorIs(t, err, assert.AnError)
-	})
-
 	t.Run("fail when pvc already exists and retrier timeouts", func(t *testing.T) {
 		// given
 		doguResource := readLdapDoguResource(t)
 		dogu := readLdapDogu(t)
-		dogu.Volumes = nil
 
 		mockClient := extMocks.NewK8sClient(t)
 		key := doguResource.GetObjectKey()
-		key.Name = doguResource.GetReservedPVCName()
-		mockClient.EXPECT().Get(mock.Anything, key, &v1.PersistentVolumeClaim{}).RunAndReturn(func(ctx context.Context, name types.NamespacedName, object client.Object, option ...client.GetOption) error {
+		mockClient.EXPECT().Get(context.TODO(), key, &v1.PersistentVolumeClaim{}).RunAndReturn(func(ctx context.Context, name types.NamespacedName, object client.Object, option ...client.GetOption) error {
```

Comment:
```suggestion
		mockClient.EXPECT().Get(context.Background(), key, &v1.PersistentVolumeClaim{}).RunAndReturn(func(ctx context.Context, name types.NamespacedName, object client.Object, option ...client.GetOption) error {
```
Use Backgroud because although TODO and Background are currently equal, they might not be in the future.